src/pages/PrivacyPolicyPage.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import {
+    DialogContent,
+    CardContent,
+    makeStyles,
+} from "@material-ui/core";
+import PrivacyPolicyContent from "../components/Offers/New/form-components/Policies/PrivacyPolicyContent";
+import { useMobile } from "../utils/media-queries";
+
+const useStyles = makeStyles((theme) => ({
+    content: {
+        padding: theme.spacing(4),
+    },
+}));
+
+const PrivacyPolicyPage = () => {
+    const classes = useStyles();
+    const isMobile = useMobile();
+    const ContentComponent = isMobile ? DialogContent : CardContent;
+
+    return (
+        <div className={classes.content}>
```

Comment:
I think this is messing up with the default margins. You should remove it (and remove the useSyles, useMobile, etc.). The code should be like this:
```js
return (
    <>
        <PrivacyPolicyContent />
    </>
);
```

Also, change the tests accordingly.